docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,123 @@
+=============
+Accessibility
+=============
+
+The Django project is committed to being accessible to everyone
+in all possible ways.
+We work to conform to established accessibility standards,
+and also to meet our own targets for supporting specific input devices
+and assistive technologies.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+We work to conform with the `Web Content Accessibility Guidelines`_ (WCAG),
+version 2.2, at the AA level.
+WCAG is the most established standard for web accessibility.
+AA-level guidelines are the most common legal compliance target worldwide.
+
+We also attempt to follow other best practices, such as:
+
+* WCAG 2.2 AAA-level guidelines.
+* The upcoming `WCAG 3.0`_ guidelines.
+* `Authoring Tool Accessibility Guidelines`_ (ATAG) 2.0 – guidelines
+  for software and services that are used to produce web content.
+
+To learn more about accessibility without diving straight into standards,
+we recommend `The A11Y Project`_, a community-driven effort
+to make digital accessibility easier.
+
+.. _Web Content Accessibility Guidelines: https://www.w3.org/TR/WCAG22/
+.. _WCAG 3.0: https://www.w3.org/TR/wcag-3.0/
+.. _Authoring Tool Accessibility Guidelines: https://www.w3.org/TR/ATAG20/
+.. _The A11Y Project: https://www.a11yproject.com/
+
+.. _accessibility-support-targets:
+
+Accessibility support targets
+=============================
+
+Beyond standards, we want to make sure Django actually works for people using
+a wide range of input devices and assistive technologies. This has benefits
+to make sure Django works better `for everyone`_.
+The best way to do this effectively is to take accessibility considerations
+into account as part of designing features. If in doubt, consult
+with users who rely on assistive technologies or with accessibility experts.
+
+.. _for everyone: https://en.wikipedia.org/wiki/Curb_cut_effect
+
+.. _accessibility-testing-baseline:
+
+Accessibility testing baseline
```

Comment:
```suggestion
Testing baseline
```

Reply:
Even if someone lands in a heading, the topic of the document is "Accessibility". Having this prefixed on every section title feels redundant and I think it's fair to expect people to be able to scroll up and down for context if they need it.